app/src/main/java/io/branch/branchster/SplashActivity.java

```diff
@@ -58,10 +59,16 @@ public void onInitFinished(BranchUniversalObject branchUniversalObject, LinkProp
                 if (branchUniversalObject == null) {
                     proceedToAppTransparent();
                 }
+                /* In case the clicked link has $android_deeplink_path the Branch will launch the MonsterViewer automatically since AutoDeeplinking feature is enabled.
+                 * Launch Monster viewer activity if a link clicked without $android_deeplink_path
+                 */
                 else if (!branchUniversalObject.getMetadata().containsKey("$android_deeplink_path")) {
```

Comment:
Doesn't this mean that deferred deep linking (through install) won't work unless $android_deeplink_path is specified?